src/controllers/pallets/PalletsNominationPoolsController.ts

```diff
@@ -0,0 +1,80 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+
+import { PalletsNominationPoolService } from '../../services';
+import AbstractController from '../AbstractController';
+
+/**
```

Comment:
These docs are actually no longer necessary they were a legacy thing before we adopted the openapi docs.